ocp_resources/resource.py

```diff
@@ -654,13 +655,21 @@ def update_replace(self, resource_dict):
         self.api.replace(body=resource_dict, name=self.name, namespace=self.namespace)
 
     @staticmethod
-    def _retry_cluster_exceptions(func):
+    def _retry_cluster_exceptions(
+        func,
+        **kwargs,
```

Comment:
What I seggested is:
```python
def _retry_cluster_exceptions(func, exceptions_dict, **kwargs)
```
func and exceptions_dict belong to this function while kwargs pass to TimeoutSampler as function kwargs.

@rnetser

Reply:
ok, I misunderstood then. I will address the comment from @rnetser accordingly. Thanks for the clarification @myakove .